shared/cmd/helpers.go

```diff
@@ -42,3 +43,16 @@ func ConfirmAction(actionText string, deniedText string) (bool, error) {
 
 	return confirmed, nil
 }
+
+func EnterPassword() (string, error) {
```

Comment:
This also is a good unit test candidate

Reply:
Good point!

Reply:
+1 unit test would be good for this

Reply:
Writing a test for `terminal.ReadPassword` is rather complicated, made a tracking issue at #5749